model/jpa/src/main/java/org/keycloak/models/jpa/JpaRealmProvider.java

```diff
@@ -152,6 +152,16 @@ public Stream<RealmModel> getRealmsStream() {
         return getRealms(query);
     }
 
+    @Override
+    public Stream<RealmModel> getRealmsStream(String search) {
```

Comment:
We missed the opportunity to add an optional `limit` parameter, to fetch only N records

Reply:
The result is a stream, so limiting the result is an implicit functionality of streams.